neuron/src/app/Neuron/CLI/New.hs

```diff
@@ -65,12 +65,12 @@ newZettelFile NewCommand {..} = do
       getEnvNonEmpty "EDITOR" >>= \case
         Nothing ->
           die "\n-e option can only be used with EDITOR environment variable set"
-        Just editorExe ->
-          pure $ editAction editorExe
-    editAction editorExe path = do
+        Just editor ->
+          pure $ editAction editor
+    editAction editor path = do
       -- Show it first in case the editor launch fails
       showAction path
-      executeFile editorExe True [path] Nothing
+      executeFile "bash" True ["-c", editor ++ ' ' : path ] Nothing
```

Comment:
Wouldn't this break if `path` had whitespace in it?

Reply:
does path not create escape characters? How was it working before?